src/gui/widgets/text_box_base.hpp

```diff
@@ -147,6 +147,13 @@ class text_box_base : public styled_widget
 		return text_.text();
 	}
 
+	std::string plain_text()
+	{
+		char* plain_text = nullptr;
+		pango_parse_markup(text().c_str(), text().size(), 0, NULL, &plain_text, nullptr, nullptr);
```

Comment:
`nullptr` please

Reply:
Also, is this name correct? It says plain text, but it's parsing markup.

Reply:
> Also, is this name correct? It says plain text, but it's parsing markup.

It's actually stripping markup from the text (for which the marked up text needs to be parsed.)